examples/big-numbers/README.md

```diff
@@ -0,0 +1,260 @@
+# Big numbers
+Ethereum uses big numbers (also known as "bignums" or "arbitrary-precision integers") to represent certain values in its codebase and in blockchain transactions. This is necessary because Ethereum uses a 256-bit numbering system, which is much larger than the number systems typically used in modern computers (such as base-10 or base-2). Using big numbers allows Ethereum to handle very large numbers, such as those that may be required to represent very large balances or quantities of a particular asset.
```

Comment:
> Ethereum uses a 256-bit numbering system, which is much larger than the number systems typically used in modern computers (such as base-10 or base-2).

Ethereum ([the EVM](https://ethereum.org/en/developers/docs/evm/)) operates on a *256-bit word size*, which is different from the usual 32-bit or 64-bit of modern machines (this has nothing to do with base-2 or base-10 which is how we choose to represent numbers). This was chosen for the ease of use with 256-bit cryptography (such as Keccak-256 hashes or secp256k1 signatures). (taken from [here](https://ethereum.org/en/developers/docs/evm/#evm-instructions))